go/cmd/controller/controller.go

```diff
@@ -48,7 +50,10 @@ func main() {
 	}
 
 	cluster := controller.NewCluster(clientset)
-	as := autoscaler.New(cluster)
+
+	as := autoscaler.New(cluster,
+		func(as *autoscaler.Autoscaler) { as.maxLoadDesired = maxLoadDesired })
```

Comment:
Does this line compile? `maxLoadDesired` is not visible outside the package.

One way is to add this function inside the `autoscaler` package:

```go
package autoscaler

func WithMaxLoad(maxLoad float64) func(a *Autoscaler) {
  a. maxLoadDesired = maxLoad
}
```

Reply:
No. Will update soon to fix this.